src/react/components/common/assetPreview/videoAsset.test.tsx

```diff
@@ -55,6 +55,18 @@ describe("Video Asset Component", () => {
         expect(wrapper.state().loaded).toBe(false);
     });
 
+    it("does not subscribe to state change callback when autoPlay is false", () => {
+        const testProps: IVideoAssetProps = { ...defaultProps,
+            autoPlay: false,
+        };
+        wrapper = createComponent(testProps);
+        mockLoaded();
+
+        const newAsset = MockFactory.createVideoTestAsset("new-video-asset");
```

Comment:
why create another asset when defaultProps already include a video asset?

Reply:
To be explicit and clear what the test was doing, so people wouldn't have to go look up what default props was to figure out where this magic asset came from, along with ensuring a second asset was created.